admin/sections/config/providers/shared.tsx

```diff
@@ -14,22 +14,33 @@ import {
 } from "@/components/ui/select";
 import { UseFormReturn } from "react-hook-form";
 
+const DEFAULT_TITLE = "Model";
+const DEFAULT_DESCRIPTION =
+  "Select a model to chat with. If you are not sure, leave it as is.";
+const DEFAULT_MODEL_FIELD_NAME = "model";
+
 export const ModelForm = ({
   form,
+  title,
+  description,
+  model_field_name,
   defaultValues,
   supportedModels,
 }: {
   form: UseFormReturn;
+  title?: string;
+  description?: string;
+  model_field_name?: string;
```

Comment:
FormField's call this just `name` - how about?
```suggestion
  name?: string;
```